README.md

```diff
@@ -10,6 +10,12 @@
     <a href="https://badge.fury.io/py/featuretools-tsfresh-primitives" target="_blank">
         <img src="https://badge.fury.io/py/featuretools-tsfresh-primitives.svg?maxAge=2592000" alt="PyPI Version" />
     </a>
+    <a href="https://badge.fury.io/py/featuretools-tsfresh-primitives" target="_blank">
+        <img src="https://badge.fury.io/py/featuretools-tsfresh-primitives.svg?maxAge=2592000" alt="PyPI Version" />
+    </a>
```

Comment:
This appears to be a duplicate of the link right above this one.

Reply:
removed